lib/run-status.js

```diff
@@ -134,9 +135,10 @@ RunStatus.prototype.prefixTitle = function (file) {
 	var separator = ' ' + chalk.gray.dim(figures.pointerSmall) + ' ';
 
 	var prefix = path.relative('.', file)
-		.replace(this.base, '')
+		.replace(new RegExp('^' + escapeStringRegexp(this.base)), '')
```

Comment:
How about:

```js
var prefix = path.relative('.', file);
if (prefix.indexOf(this.base) === 0) {
  prefix = prefix.slice(this.base.length);
}

prefix = prefix
  .replace(/\.spec/, '')
  // etc
```

Saves us a dependency and (to my eyes) is more obvious than the regular expression.